polyply/src/load_library.py

```diff
@@ -13,75 +13,109 @@
 # limitations under the License.
 
 import os
-import pathlib
+from pathlib import Path
 import vermouth
 from vermouth.gmx.rtp import read_rtp
 from vermouth.citation_parser import read_bib
+from vermouth.log_helpers import StyleAdapter, get_logger
 from polyply import DATA_PATH
 from .ff_parser_sub import read_ff
+from .build_file_parser import read_build_file
 from .polyply_parser import read_polyply
 
+LOGGER = StyleAdapter(get_logger(__name__))
+BUILD_FILE_PARSERS = {'bld': read_build_file}
 FORCE_FIELD_PARSERS = {'rtp': read_rtp, 'ff': read_ff, 'itp': read_polyply, 'bib': read_bib}
 
+def determine_parser(file_, file_parsers):
```

Comment:
let's call this thing `get_parser` or something along those lines to keep it a bit shorter